src/FINDOLOGIC/Export/Helpers/DataHelper.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+namespace FINDOLOGIC\Export\Helpers;
+
+class EmptyValueNotAllowedException extends \RuntimeException
+{
+    public function __construct()
+    {
+        parent::__construct('Empty values are not allowed!');
+    }
+}
+
+class DataHelper
+{
+    /**
+     * Checks if the provided value is empty.
+     *
+     * @param string $value The value to check.
+     * @throws EmptyValueNotAllowedException If the value is empty.
+     * @return string Returns the value if not empty.
+     */
+    public static function emptyValueCheck($value)
```

Comment:
Prefer imperative-style naming for methods, such as `checkForEmptyValue` or `assertNotEmpty`. The latter has the advantage that it indicates that being non-empty is an absolute must-have.

Reply:
Done